tests/bug-2917/test.yaml

```diff
@@ -0,0 +1,215 @@
+requires:
+  min-version: 7
+
+pcap: false
+
+args:
+  - --engine-analysis
```

Comment:
@victorjulien does anything seem out of the ordinary in the following checks per rule?